exercise_perseus_render/assets/src/vue/index.vue

```diff
@@ -1,23 +1,59 @@
 <template>
 
   <div id="exercise-container">
-    <perseus v-if="item" :item="item"></perseus>
+    <assessment-wrapper
+      v-ref:wrapper
+      v-if="exercise"
+      :itemId="itemId"
+      :masteryModel="exercise.mastery_model"
+      :masterySpacingTime="exercise.masterySpacingTime">
```

Comment:
"HTML is case-insensitive, remember to use kebab-case for props in templates" Vue bites